STYLE_GUIDE.md

```diff
@@ -0,0 +1,88 @@
+# Style Guide
+
+This document outlines the style guide for the `epinowcast` package. This guide is a work in progress and will be updated as the package evolves. We welcome contributions to this guide and encourage you to raise issues or submit PRs if you have any suggestions.
+
+In addition to this guide we also follow the [tidyverse style guide](https://style.tidyverse.org/). This guide is a subset of the `tidyverse` style guide and outlines the additional style requirements for the `epinowcast` package.
+
+## Naming conventions
+
+- We use a `enw_` prefix to delineate functions that are part of the `epinowcast` package. This is to avoid conflicts with other packages and to make it clear to users which functions are part of the package.
```

Comment:
```suggestion
- We use a `enw_` prefix to delineate functions that are exported by the `epinowcast` package. This is to avoid conflicts with other packages and to make it clear to users which functions are part of the package.
```